tests/scripts/all.sh

```diff
@@ -592,6 +583,15 @@ msg "test: allow SHA1 in certificates by default"
 make test
 tests/ssl-opt.sh -f SHA-1
 
+msg "build: default config, MBEDTLS_RSA_NO_CRT, make, gcc"
+cleanup
+cp "$CONFIG_H" "$CONFIG_BAK"
+scripts/config.pl set MBEDTLS_RSA_NO_CRT
+CC=gcc CFLAGS='-Werror -Wall -Werror -O0' make
```

Comment:
Because `make` is now a function in keep-going mode, `make` must come first on the command line:

    make CC=gcc CFLAGS='-Werror -Wall -Werror -O0'

Reply:
Fixed in d8b3851.